hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/ScmConfigKeys.java

```diff
@@ -372,6 +372,13 @@
       "ozone.scm.pipeline.per.metadata.disk";
 
   public static final int OZONE_SCM_PIPELINE_PER_METADATA_VOLUME_DEFAULT = 2;
+
+  public static final String OZONE_DATANODE_METADATA_VOLUME_FREE_SPACE_MIN =
+      "ozone.scm.datanode.metadata.volume.free-space.min";
```

Comment:
There is a "hdds.metadata.dir" property which specify a directory to save datanode whole metadata. In order not to confuse with this meta directory,  I would suggest we use "ratis" instead here, like "ozone.scm.datanode.ratis.volume.free-space.min"

Reply:
Sure, we could use the suggested name.